addons/addon-base-raas-ui/packages/base-raas-ui/src/models/environments-sc/ScEnvironmentsStore.js

```diff
@@ -130,6 +157,22 @@ const ScEnvironmentsStore = BaseStore.named('ScEnvironmentsStore')
       return self.environments.get(id);
     },
 
+    canChangeState(id) {
+      const outputs = self.environments.get(id).outputs;
+      let result = false;
+      // TODO: Remove isRStudio check after CNAME patch
+      let isRStudio = false;
+      outputs.forEach(output => {
+        if (output.OutputKey === 'Ec2WorkspaceInstanceId' || output.OutputKey === 'NotebookInstanceName') {
```

Comment:
Since you're explicitly checking for RStudio in the next if block, I think you can safely remove the check for 'Ec2WorkspaceInstanceId' here